resources/public/api/conf/1.0/common/errors.yaml

```diff
@@ -0,0 +1,94 @@
+components:
+  examples:
+
+    # 400
+
+    formatNino:
+      description: The format of the supplied NINO field is not valid.
+      value:
+        code: FORMAT_NINO
+        message: The provided NINO is invalid
+
+    formatTaxYear:
+      description: The format of the supplied taxYear field is not valid.
+      value:
+        code: FORMAT_TAX_YEAR
+        message: The provided tax year is invalid
+
+    formatSource:
+      description: The format of the supplied Source field is not valid.
+      value:
+        code: FORMAT_SOURCE
+        message: The provided source is invalid
+
+    formatValue:
+      description: One or more values have been added with the incorrect format.
+      value:
+        code: FORMAT_VALUE
+        message: The field should be between 0 and 99999999999.99
+
+    formatCustomerRef:
+      description: The format of customer reference is not valid.
+      value:
+        code: FORMAT_CUSTOMER_REFERENCE
+        message: The provided customer reference is not valid
+
+    ruleIncorrectOrEmptyBody:
+      description: An empty or non-matching body was submitted.
+      value:
+        code: RULE_INCORRECT_OR_EMPTY_BODY_SUBMITTED
+        message: An empty or non-matching body was submitted
+
+    ruleTaxYearNotSupported:
+      description: |
+        The specified tax year is not supported.
+        Either the tax year specified is before the minimum tax year value,
+        or it is after the maximum tax year value.
+      value:
+        code: RULE_TAX_YEAR_NOT_SUPPORTED
+        message: The tax year specified does not lie within the supported range
+
+    ruleTaxYearRangeInvalid:
+      description: Tax year range invalid. A tax year range of one year is required.
+      value:
+        code: RULE_TAX_YEAR_RANGE_INVALID
+        message: Tax year range invalid. A tax year range of one year is required
+
+    ruleTaxYearNotEnded:
+      description: The submission has been made before the Tax year it relates to has ended.
+      value:
+        code: RULE_TAX_YEAR_NOT_ENDED
+        message: Tax year not ended
+
+
+    # 403
+
+    clientOrAgentNotAuthorised:
+      description: |
+        The client and/or agent is not authorised. This is normally because:
```

Comment:
The description should be `The client or agent is not authorised. This is because: the client is not subscribed to MTD, the agent is not subscribed to Agent Services, or the client has not authorised the agent to act on their behalf.` and the error should be `The client or agent is not authorised`

Reply:
Did I miss something? That's exactly what they are?

Reply:
Just a couple words difference (i.e. first sentence yaml says `and/or` spec says `or`, second sentence yaml says `This is normally because` spec says `This is because`)